jcl/src/java.base/share/classes/java/lang/invoke/VarHandle.java

```diff
@@ -1090,7 +1101,7 @@ public static VarHandleDesc ofArray(ClassDesc arrayClass) throws NullPointerExce
 				/*[MSG "K0625", "{0} is not an array type."]*/
 				throw new IllegalArgumentException(com.ibm.oti.util.Msg.getString("K0625", arrayClass.descriptorString())); //$NON-NLS-1$
 			}
-			return new VarHandleDesc(Kind.ARRAY, arrayClass, "_", arrayClass.componentType()); //$NON-NLS-1$
+			return new VarHandleDesc(Kind.ARRAY, null, "_", arrayClass); //$NON-NLS-1$
```

Comment:
The constructor says it will throw an NPE for any null args.

Also, it says the name should be `(blank for array)`.  Does that mean `"_"` or `""` (empty string)?  Its not clear which is correct to me